core/trino-main/src/main/java/io/trino/metadata/OutputTableHandle.java

```diff
@@ -24,48 +22,14 @@
 
 import static java.util.Objects.requireNonNull;
 
-public final class OutputTableHandle
+public record OutputTableHandle(CatalogHandle catalogHandle, SchemaTableName tableName, ConnectorTransactionHandle transactionHandle, ConnectorOutputTableHandle connectorHandle)
```

Comment:
Are we excluding  SchemaTableName from hashCode & equals intentionally? Or is it oversight in https://github.com/trinodb/trino/pull/12869?

Reply:
Seems like a omission. Let me remove this equals and hashCode in the separate commit.